tests/30-main.spec.js

```diff
@@ -254,4 +255,450 @@ describe('main', () => {
       credential, documentLoader, verifyRevocationListCredential: false});
     result.verified.should.equal(false);
   });
+
+  it('should fail to verify status on missing credential param', async () => {
+    let err;
+    let result;
+    try {
+      result = await checkStatus({
+        documentLoader, verifyRevocationListCredential: false});
+      result.verified.should.equal(false);
+    } catch(e) {
+      err = e;
+    }
+    should.not.exist(err);
+    should.exist(result);
+    result.should.be.an('object');
+    result.should.have.property('verified');
+    result.verified.should.be.a('boolean');
+    result.verified.should.be.false;
+    result.should.have.property('error');
+    result.error.should.be.instanceof(TypeError);
+    result.error.message.should.contain('"credential" must be an object');
+  });
+
+  it('should fail to verify if credential is not an object for ' +
+   'statusTypeMatches"', async () => {
+    let err;
+    let result;
+    try {
+      result = statusTypeMatches('{credential}');
```

Comment:
This test is a little odd -- it seems to be testing that the options passed into `statusTypeMatches` are an object, not that the named param `credential` is not an object. I've made a suggestion below to change it to match what the test says:

```suggestion
      result = statusTypeMatches({credential: ''});
```